src/components/calcite-select/calcite-select.e2e.ts

```diff
@@ -0,0 +1,166 @@
+import { newE2EPage } from "@stencil/core/testing";
+import { accessible, focusable, reflects, renders } from "../../tests/commonTests";
+
+describe("calcite-select", () => {
+  function createSimpleTestMarkup(): string {
```

Comment:
I am curious: is there a reason to make this a function? As opposed to something like:

```
const simpleTest = `
  <calcite-select label="required-for-a11y-test">
    <calcite-option>uno</calcite-option>
    <calcite-option>dos</calcite-option>
    <calcite-option>tres</calcite-option>
  </calcite-select>
`;

it("renders", async () => renders(simpleTest));
```

Like do the test helpers mutate the string you pass?